giskard-ml-worker/ml_worker/testing/performance_tests.py

```diff
@@ -49,6 +49,8 @@ def test_auc(self, actual_slice: GiskardDataset, model: GiskardModel, threshold=
             ))
 
     def _test_classification_score(self, score_fn, gsk_dataset: GiskardDataset, model: GiskardModel, threshold=1):
+        if not gsk_dataset.target:
```

Comment:
Could add the same check for the rest of the performance tests that are missing it:
- test_auc
- _test_regression_score

Reply:
done the needful